js/src/lib/Details/Markdown.js

```diff
@@ -26,8 +26,19 @@ const renderAndEscapeMarkdown = ({ source, githubRepo }) => {
         path,
       });
 
+    // manually ask for sanitation of svgs, otherwise it will have wrong content-type
+    const sanitize = href =>
+      `${href}${href.indexOf('//') === -1 &&
+      String.prototype.endsWith &&
+      href.endsWith('.svg')
+        ? '?sanitize=true'
+        : ''}`;
```

Comment:
This is very hard to read, since it contains a multiline template string which contains a ternary. Can you please make it a bit clearer?  For example, something like:
```
function sanitize(href) {
  if (!href.endsWith('.svg')) {
    return href;
  }
  return href + '?sanitize=true';
}
```


Also, shouldn't this only be done for relative links? Won't this current implementation add `?sanitize=true` for all SVG links, even if they're not GitHub links?

Reply:
it was already done only on relative links, made the conditional clearer now though